ethereum/balancer_v1/view_liquidity.sql

```diff
@@ -58,7 +58,7 @@ CREATE MATERIALIZED VIEW balancer_v1.view_liquidity AS (
             1,
             2
         HAVING
-            SUM(sample_size) > 3
+            SUM(sample_size) > 5
```

Comment:
same comment here on this file with same user generated table restricting the rebuild